docs/beta/beta-dashboard-data-API.md

```diff
@@ -0,0 +1,354 @@
+---
+id: beta-dashboard-data-API
```

Comment:
I'd get rid of "beta" from the id because this will change once it goes GA

```suggestion
id: dashboard-data-API
```

Reply:
This is to stay as it is. Verified by Arpit.

Reply:
Sent you a Slack message -- let's meet with him to go over our style